src/Sql/dbo/Stored Procedures/OrganizationUser_ReadByRole.sql

```diff
@@ -0,0 +1,15 @@
+CREATE PROCEDURE [dbo].[OrganizationUser_ReadByRole]
```

Comment:
Let's also rename this to match it's intent and function since it's not using `=` but instead is using `<=` where role is a function of higher access in descending order of numerical value.

```suggestion
CREATE PROCEDURE [dbo].[OrganizationUser_ReadByMinimumRole]
```